src/containers/LogIn/index.js

```diff
@@ -45,39 +53,41 @@ const Login = () => {
   };
 
   return (
-    <Box containerClassName="login__box">
-      <div className="box__title">
-        NeoPosteos
-      </div>
-      <form onSubmit={handleSubmit(onSubmitHandler)}>
-        <Input
-          placeholder="Email"
-          id="email"
-          type="email"
-          {...register('email', { required: true })}
-        />
-        <Input
-          placeholder="Password"
-          id="password"
-          type="password"
-          {...register('password', { required: true })}
-        />
-        <div className="submit-btn">
-          <Button
-            text="SignUp"
-            type="submit"
-          />
-        </div>
-        <div className="signup-link">
-          <p>
-            Don&apos;t have an account?
-            Shame on you!
-            <br />
-            <Link to="/signup">Sign Up</Link>
-          </p>
+    <div className="login-container">
```

Comment:
No need for container in here as the BLOCK of BEM
```suggestion
    <div className="login">
```